src/NUnitFramework/tests/Assertions/AssertThrowsAsyncTests.cs

```diff
@@ -223,6 +224,7 @@ public void DerivedExceptionThrown()
         }
 
         [Test]
+        [SetUICulture("en-US")]
         public void DerivedExceptionThrownAsync()
```

Comment:
Failure reason: Test asserts a constant English exception message that doesn't match the localized exception message.